### ISSUE TITLE
Libusb included in Repo

### ISSUE THREAD

**Issue by xmosbrendon** [xmos/lib_device_control#55]
This repo has a copy of libusb. Our tools ship with a different version of libusb. We need to separate the libusb dependancy from the repo and either have another repo or simply use the tools copy

**Comment by humphrey-xmos** [comment]
Agreed, but the tools do not ship with a libusb header file. I've asked about including this.
